app/models/user.js

```diff
@@ -0,0 +1,31 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
```

Comment:
for convention, model's name must be singular.

Reply:
ok